sq/src/commands/key.rs

```diff
@@ -304,6 +307,22 @@ fn extract_cert(config: Config, m: &ArgMatches) -> Result<()> {
     Ok(())
 }
 
+fn extract_sdkms(config: Config, m: &ArgMatches) -> Result<()> {
+    let mut output = config.create_or_stdout_safe(m.value_of("output"))?;
+
+    let key = match m.value_of("sdkms-key") {
+        Some(key_name) => sdkms::extract_tsk_from_sdkms(key_name)?,
+        None => unreachable!("name is compulsory")
+    };
+
+    if m.is_present("binary") {
+        key.as_tsk().serialize(&mut output)?;
+    } else {
+        key.as_tsk().armored().serialize(&mut output)?;
+    }
+    Ok(())
```

Comment:
do we want to return something here in Ok()?

Reply:
Not really, since this function is an analog of the existing `extract_cert` with the same signature. The certificate is output to sdtout, or a file.